example/49_maxpool2d_bwd/maxpool2d_bwd_common.hpp

```diff
@@ -0,0 +1,224 @@
+// SPDX-License-Identifier: MIT
+// Copyright (c) 2018-2023, Advanced Micro Devices, Inc. All rights reserved.
+
+#pragma once
+
+#include <iostream>
+
+#include "ck/ck.hpp"
+#include "ck/utility/reduction_enums.hpp"
+#include "ck/utility/reduction_functions_accumulate.hpp"
+#include "ck/tensor_operation/gpu/device/reduction_operator_mapping.hpp"
```

Comment:
Removing the including since they are not needed

Reply:
eac452abee416c8eb5ea0383ffa868f09936686e